docs/services/vision/segmentation.md

```diff
@@ -218,6 +219,83 @@ The following parameters are available for a `detector_3d_segmenter`.
+
+This is determined by the value of the configuration attribute `with_geometries`:
+
+- `true`: this segmenter will return point clouds within the `GeometryInFrame` object it captures.
+- `false`: this segmenter will return a single point within the `GeometryInFrame` object it captures.
+
+If `true`, start configuring your segmenter by following [these instructions](/services/frame-system/#configuration) to configure the relative spatial orientation of the components of your robot within Viam's [frame system service](/services/frame-system/).
+
+After doing this, your camera will populate its own `Properties` with the intrinsic parameters of the geometric configuration.
+You can then get those parameters from your camera through the [camera API](/components/camera/#getproperties).
+
+Then, configure an `obstacles_depth` segmenter:
+
+{{< tabs >}}
+{{% tab name="JSON Template" %}}
+
+Add the vision service object to the services array in your raw JSON configuration:
+
+``` json {class="line-numbers linkable-line-numbers"}
+"services": [
+    {
+    "name": "<segmenter_name>",
+    "type": "vision",
+    "model": "obstacles_pointcloud"
+    "attributes": {
+        "h_min_m": <number>,
+        "h_max_m": <number>,
+        "theta_max_deg": <number>,
+        "return_pcds": <boolean>,
+        "with_geometries": <boolean>,
+    }
+    },
+    ... // Other services
+]
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+"services": [
+{
+    "name": "rc_segmenter",
+    "type": "vision",
+    "model": "obstacles_pointcloud",
+    "attributes": {
+        "h_min_m": 0.0,
+        "h_max_m": 1.0,
+        "theta_max_deg": 45,
+        "return_pcds": "false",
+        "with_geometries": "false"
+    }
+}
+]
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following parameters are available for a `"obstacles_depth"` segmenter:
+
+| Parameter | Inclusion | Description |
+| --------- | --------- | ----------- |
+| `return_pcds` | Required | If you would like point clouds or a single point returned within their `GeometryInFrame` object. `true` indicates wanting point clouds. <br> Example: `false` </br> |
+| `h_min_m` | Optional | The minumum vertical height in meters acceptable so that any obstacle strictly lower than is not deemed an obstacle by the segmenter. <br> Default: `0.0` </br> |
+| `h_max_m` | Optional | The maximum vertical height in meters acceptable so that any obstacle strictly higher than is not deemed an obstacle by the segmenter. <br> Default: `1.0` </br> |
+| `theta_max_deg` | Optional | The largest slope in degrees acceptable for non-obstacle surface ramps. <br> Default: `45` </br> |
+| `with_geometries` | Optional | If you have configured the geometries, or, spatial orientation of the components of your robot, within the frame system. `true` is necessary to `return_pcds` properly, if `"return_pcds": "true"`  <br> Example: `false` </br> |
```

Comment:
@kharijarrett Any suggestions here on the tables are very welcome, I'm not sure if I'm phrasing properly esp. `with_geometries`